storage/gcsemu/example_test.go

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright 2016 Google LLC
```

Comment:
should I remove this as well? copy and paste...

Reply:
yeah don't add a Google copyright to new files